compose/src/main/kotlin/com/yourssu/handy/compose/textfield/OutlinedTextField.kt

```diff
@@ -0,0 +1,126 @@
+import androidx.compose.foundation.interaction.collectIsFocusedAsState
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.BasicTextField
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.unit.dp
+import com.yourssu.handy.compose.HandyTheme
+import com.yourssu.handy.compose.Icon
+import com.yourssu.handy.compose.Text
+import com.yourssu.handy.compose.foundation.HandyTypography
+
+@Composable
+fun OutlinedTextField(
+    value: String,
+    onValueChange: (String) -> Unit,
+    modifier: Modifier = Modifier,
+    placeholder: String? = null,
+    trailingIcon: ImageVector? = null,
+    enabled: Boolean = true,
+    isError: Boolean = false,
+    isSingleLine: Boolean = true,
+    onClickTrailingIcon: () -> Unit = {}
+) {
+    val interactionSource = remember { MutableInteractionSource() }
+    val isFocused = interactionSource.collectIsFocusedAsState().value
+
+    val (borderColor, cursorColor, textColor, placeholderTextColor) = getTextFieldStyle(
+        enabled,
+        isError,
+        isFocused
+    )
+
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .clip(RoundedCornerShape(12.dp))
+            .background(HandyTheme.colors.bgBasicLight)
+            .border(1.dp, borderColor, RoundedCornerShape(12.dp))
+            .padding(start = 16.dp, end = 12.dp, top = 12.dp, bottom = 12.dp)
+    ) {
+        BasicTextField(
+            value = value,
+            onValueChange = onValueChange,
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxWidth(),
+            textStyle = HandyTypography.B1Rg16.toTextStyle().copy(color = textColor),
+            enabled = enabled,
+            singleLine = isSingleLine,
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Text),
+            interactionSource = interactionSource,
+            cursorBrush = SolidColor(cursorColor),
+            decorationBox = { innerTextField ->
+                if (value.isEmpty() && placeholder != null) {
+                    Text(
+                        text = placeholder,
+                        style = HandyTypography.B1Rg16,
+                        color = placeholderTextColor
+                    )
+                }
+                innerTextField()
+            }
+        )
+
+        trailingIcon?.let {
+            Icon(
+                imageVector = trailingIcon,
```

Comment:
```suggestion
        trailingIcon?.let {
            Icon(
                imageVector = it,
```
`it`으로 하는 게 통일성 있을 것 같아요